website/content/en/preview/concepts/node-templates.md

```diff
@@ -34,6 +34,9 @@ spec:
   metadataOptions: { ... }       # optional, configures IMDS for the instance
   blockDeviceMappings: [ ... ]   # optional, configures storage devices for the instance
   detailedMonitoring: "..."      # optional, configures detailed monitoring for the instance
+  licenseSpecifications: [ ... ] # optional, configures licenseSpecifications in aws launch templates
```

Comment:
`launchTemplateName` should be the only undocumented field since we don't want anyone to use it. It's docu-deprecated :D 

Everything else should be documented there